moban/constants.py

```diff
@@ -75,3 +75,5 @@
 JINJA_FILTER_EXTENSION = "jinja_filter"
 JINJA_TEST_EXTENSION = "jinja_test"
 JINJA_GLOBALS_EXTENSION = "jinja_globals"
+
+TEMPLATE_ENGINE_EXTENSION = "template_engine"
```

Comment:
This solves https://github.com/moremoban/moban/issues/58 ?

Reply:
Yes, it will be fixed by this PR. And template_type in moban file has been rendered to zero effect. I am thinking to give its power as default template status: unknown -> the template of user choice. I.e. Template_type: mako, then for unknown template files, mako engine will be used instead of jinja2.

Reply:
Keeping it as the default template status would be perfect IMO